adb_graphics/default_specs.yml

```diff
@@ -873,3 +886,44 @@ vvort: # Vertical vorticity
   mx02: # Hourly maximum of vertical vorticity over 0-2 km layer
     <<: *vvort 
     title: 0-2km Max Vertical Velocity (over prev hour)
+wspeed: # Wind Speed
+  10m: &wspeed
+    clevs: !!python/object/apply:numpy.arange [5, 95, 5]
+    cmap: NWS8bitVel
+    colors: wind_colors
+    ncl_name: UGRD_P0_L103_GLC0
+    ticks: 5
+    title: 10m Wind
+#    transform: [conversions.ms_to_kt, windspeed]
+    transform: [vector_magnitude, conversions.ms_to_kt]
```

Comment:
This change allows the grib test to just have the 'self' argument.

Reply:
This seems like a nice direct way to do this. If we aren't going to use the windspeed method, though, maybe we should remove it. Do you mind doing that here since you're handling the windspeed graphics?